Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -1251,6 +1259,7 @@ public void run() {
                 runOnUiThread(new Runnable() {
                     @Override
                     public void run() {
+                        AppLog.add(Type.ACCOUNT, "Token not authorized");
```

Comment:
```suggestion
                        AppLog.add(Type.ACCOUNT, "Access token not authorized");
```

Should we make this log message the same as the other statement when an access token is not authorized just for consistency?  If we want to differentiate this statement from the other, we could make it something like "Access token not authorized (NotesActivity.onUserStatusChange)" for this one and "Access token not authorized (NotesActivity.userAuthenticationIsInvalid)" for the previous one.